src/main/java/bio/terra/cli/service/utils/HttpUtils.java

```diff
@@ -54,6 +55,27 @@ private HttpUtils() {}
   public static HttpResponse sendHttpRequest(
       String urlStr, String requestType, String accessToken, Map<String, String> params)
       throws IOException {
+    Map<String, String> headers = ImmutableMap.of("accept", "*/*");
+    if (accessToken != null) {
+      headers.put("Authorization", "Bearer " + accessToken);
+    }
```

Comment:
I think this needs to be a mutable map in order to do a `put`, or you can use a builder.

Reply:
Oops, done :)